Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -46,40 +62,49 @@ public PowerShellKernel()
         {
             Name = DefaultKernelName;
             _cancellationSource = new CancellationTokenSource();
+            _cancellationSourceLock = new object();
+            _psHost = new PSKernelHost();
+            _lazyPwsh = new Lazy<PowerShell>(CreatePowerShell);
+        }
+
+        private PowerShell CreatePowerShell()
+        {
+            const string PSTelemetryEnvName = "POWERSHELL_DISTRIBUTION_CHANNEL";
+            const string PSTelemetryChannel = "dotnet-interactive-powershell";
+            const string PSModulePathEnvName = "PSModulePath";
+
+            // Set the distribution channel so telemetry can be distinguished in PS7+ telemetry
+            Environment.SetEnvironmentVariable(PSTelemetryEnvName, PSTelemetryChannel);
+
+            // Create PowerShell instance
+            var iss = InitialSessionState.CreateDefault2();
+            if(Platform.IsWindows)
+            {
+                // This sets the execution policy on Windows to RemoteSigned.
+                iss.ExecutionPolicy = ExecutionPolicy.RemoteSigned;
```

Comment:
Rob recently added a change to PSES that did a best effort to honor the ExecutionPolicy.

https://github.com/PowerShell/PowerShellEditorServices/pull/1208

We may want to do that in the kernel as well but I won't block on this.

Reply:
Can you open an issue on that?